addon/components/flexberry-maptoolbar.js

```diff
@@ -13,11 +13,20 @@ export default Ember.Component.extend({
 
   activeToolName: null,
 
+  _isMeasureTool(toolname) {
+    return toolname && toolname.substr(-11) ===  'measuretool';
```

Comment:
For all measuredtools add superclass measuredtool with common methods: isMultiTool()=true, enable(), disable(), measured(), measuredClose()
Add method isMultiTool()=false to maptool